src/shared/components/ShareThroughEmailModal/ShareThroughEmailModal.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { FunctionComponent, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import {
+	BlockHeading,
+	Box,
+	Button,
+	Flex,
+	FlexItem,
+	Modal,
+	ModalBody,
+	Spacer,
+	TextInput,
+} from '@viaa/avo2-components';
+
+import toastService from '../../../shared/services/toast-service';
+import { copyToClipboard } from '../../helpers';
+import { shareThroughEmail } from '../../helpers/share-through-email';
+
+import { EmailTemplateType } from './share-through-email.types';
+import './ShareThroughEmailModal.scss';
+
+interface AddToCollectionModalProps {
+	modalTitle: string;
+	type: EmailTemplateType;
+	emailLinkHref: string;
+	emailLinkTitle: string;
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+const ShareThroughEmailModal: FunctionComponent<AddToCollectionModalProps> = ({
+	modalTitle,
+	type,
+	emailLinkHref,
+	emailLinkTitle,
+	isOpen,
+	onClose,
+}) => {
+	const [t] = useTranslation();
+
+	const [emailAddress, setEmailAddress] = useState<string>('');
+	const [isProcessing, setIsProcessing] = useState<boolean>(false);
+
+	const copyLink = () => {
+		copyToClipboard(emailLinkHref);
+		toastService.success(t('De url is naar het klembord gekopieerd'));
+	};
+
+	const sendEmail = async () => {
+		try {
+			setIsProcessing(true);
+			await shareThroughEmail(emailAddress, emailLinkTitle, emailLinkHref, type);
+			toastService.success(t('De email is verstuurd'));
+		} catch (err) {
+			console.error('Failed to send email to share item', err, {
+				emailAddress,
+				emailLinkTitle,
+				emailLinkHref,
+				type,
+			});
+			toastService.danger('Het versturen van de email is mislukt');
+		}
+		setIsProcessing(false);
+	};
+
+	return (
+		<Modal
+			className="m-share-through-email-modal"
+			title={modalTitle}
+			size="medium"
+			isOpen={isOpen}
+			onClose={onClose}
+			scrollable
+		>
+			<ModalBody>
+				<BlockHeading type="h4">
+					<Trans>Kopieer deze publieke link</Trans>
+				</BlockHeading>
+				<p>
+					<Trans>
```

Comment:
Je kan dit doen met die Trans component: `<Trans parent="p">...</Trans>` dan wordt die gewrapped in p tags